Libraries/PeriphDrivers/Include/MAX32657/i3c.h

```diff
@@ -153,9 +153,9 @@ struct _i3c_ccc_req_t {
 };
 
 /**
- * @brief Private SDR/I2C request structure.
+ * @brief Private SDR request structure.
  *
- * SDR/I2C read and write to an I3C target.
+ * SDR read and write to an I3C target.
```

Comment:
Not critical but `SDR/I2C ...` is the correct comment.

Reply:
Feel free to push a new commit to fix it.